File: cell.py
class Cell ():
    def __init__(self, **kwargs):
        self.x = kwargs.get( 'x', 0 )
        self.y = kwargs.get( 'y', 0 )
        self.coord = kwargs.get('coord', [ self.x, self.y ])
        self.visited = kwargs.get('visited', 'false')
        for key, value in kwargs.items():
            setattr(self, key, value)

    def __str__():
        return "Cell, coord: {}, visited: {}".format( coord, visited )

File: js/objects/cell.js
function Cell(kwargs) {

    if (!kwargs) {
        kwargs = {};
    }

    this.name =     kwargs.name     || 'Cell';
    this.desc =     kwargs.desc     || 'Contains data about a point on a grid';  
    this.coord =    kwargs.coord    || [0,0];
    this.visited =  kwargs.visited  || 'false';   

    for (var k in kwargs) {
        if (kwargs.hasOwnProperty(k)) {
            this[k] = kwargs[k] || '';
        }
    }

    this.testMethod =   function () {
        console.log("testMethod called");
    }
}

Cell.prototype.getInfo = function () {
    return this.name + ', coord: ' + this.coord + ', visited: ' + this.visited;
}

Cell.prototype.__init__